nodewatcher/core/generator/cgm/models.py

```diff
@@ -398,6 +398,7 @@ def get_index(self):
         return list(vifs).index(self.pk)
 
 registration.point('node.config').register_choice('core.interfaces#wifi_mode', registration.Choice('mesh', _("Mesh")))
+registration.point('node.config').register_choice('core.interfaces#wifi_mode', registration.Choice('802.11s', _("802.11s Mesh")))
```

Comment:
Perhaps we should rename both mesh modes as follows:
* For existing ad-hoc, we can change the description to `"Mesh (ad-hoc)"` and leave the `mesh` identifier.
* For the new 802.11s, give it identifier `mesh11s` and change the description to `"Mesh (802.11s)"`.